src/app/static/src/app/components/ErrorReport.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <modal :open="open">
+        <h4 v-translate="'reportIssues'"></h4>
+        <form class="form">
+            <div class="form-row">
+                <div class="col-4">
+                    <label for="section" v-translate="'section'"></label>
+                </div>
+                <div class="col-8">
+                    <select class="form-control"
+                            v-model="currentSection"
+                            id="section">
+                        <option v-for="step in steps"
+                                :key="step.number"
+                                v-translate:value="step.textKey"
```

Comment:
Nice. I wouldn't have thought of this, but it makes sense if we're assuming that if the interface is currently in (e.g.) French then the user will be writing the error report in French, and the step name should therefore also be submitted translated so that the whole report is in a single language.

Reply:
I guess it depends if were seeing this value as text or more of an enum. Might be easier to do analytics on the logged error reports by section if these values were the same across languages (as well as easier on anglophones dealing with the error reports). But consistency is good too!

Reply:
Good shout, will use keys